pkgs/applications/networking/browsers/mozilla-plugins/mplayerplug-in/default.nix

```diff
@@ -18,7 +18,8 @@ stdenv.mkDerivation rec {
      else "");
 
   buildInputs = [ pkgconfig browser (browser.gtk) libXpm gettext ];
-  
+  propagatedUserEnvPkgs = [ mplayer ];
```

Comment:
Even if not, surely this change doesn't make it worse. :) I saw a FIXME (see `all-packages.nix`) and fixed it. I didn't use the thing for quite a while.